package.json

```diff
@@ -65,4 +65,4 @@
     "webpack": "^1.14.0",
     "yargs": "^6.6.0"
   }
-}
+}
```

Comment:
Looks like this just removed the trailing newline, which you probably didn't intend.